weather_sp/README.md

```diff
@@ -75,17 +77,39 @@ On the other hand, to specify a specific pattern use
 --input-pattern 'gs://test-tmp/era5/2017/*/*.nc'
 ```
 
-## Output
+## Output & Dimensions to split
```

Comment:
I initially tried using a separate list for split dimensions and the formatting, but gave up about 2 min after I had a prototype and got the first error because I modified one but not the other...